### ISSUE TITLE
Doesn't work with rollup API

### ISSUE THREAD

**Issue by nghuuphuoc** [rollup/rollup-plugin-multi-entry#8]
It works from the command line (`rollup -c`) with the configuration file as

```javascript
import multiEntry from 'rollup-plugin-multi-entry'

export default {
    entry: 'src/js/test/**.js',
    external: ['jQuery'],
    plugins: [multiEntry()],
    format: 'umd',
    dest: 'dist/js/myBundle.umd.js',
    globals: {
        'jQuery': 'jQuery'
    },
    moduleName: 'MyBundle'
}
```

But when trying to use the API (I am creating a gulp task):

```javascript
var multiEntry = require('rollup-plugin-multi-entry');

gulp.task('bundle-umd', function() {
    return rollup.rollup({
        entry: 'src/js/test/**.js',
        external: ['jQuery'],
        plugins: [multiEntry]
    }).then(function(bundle) {
        return bundle.write({
            format: 'umd',
            dest: 'dist/js/myBundle.umd.js',
            moduleName: 'myBundle',
            globals: {
                'jQuery': 'jQuery'
            }
        });
    });
});
```

Running `gulp bundle-umd` gives me the error:

```
[13:44:38] Using gulpfile .../gulpfile.js
[13:44:38] Starting 'bundle-umd'...
[13:44:38] 'bundle-umd' errored after 3.03 ms
[13:44:38] Error: Could not load null: path must be a string
    at /node_modules/rollup/dist/rollup.js:7361:10
    at runMicrotasksCallback (node.js:337:7)
    at process._tickCallback (node.js:355:11)
    at Function.Module.runMain (module.js:503:11)
    at startup (node.js:129:16)
    at node.js:814:3
```

Do I miss anything?

**Comment by TrySound** [comment]
@nghuuphuoc Yup.
```js
var multiEntry = require('rollup-plugin-multi-entry').default;
```

@eventualbuddha This really sucks.Why we even need additional named export?

**Comment by nghuuphuoc** [comment]
@TrySound 
Using `var multiEntry = require('rollup-plugin-multi-entry').default;` doesn't work.
I got the same error:
```
Error: Could not load null: path must be a string
    at /node_modules/rollup/dist/rollup.js:7361:10
    at runMicrotasksCallback (node.js:337:7)
    at process._tickCallback (node.js:355:11)
    at Function.Module.runMain (module.js:503:11)
    at startup (node.js:129:16)
    at node.js:814:3
```

**Comment by nghuuphuoc** [comment]
Updated:
```javascript
var multiEntry = require('rollup-plugin-multi-entry').default;  // ------> Updated

gulp.task('bundle-umd', function() {
    return rollup.rollup({
        entry: 'src/js/test/**.js',
        external: ['jQuery'],
        plugins: [multiEntry()] /// -----> Updated
    }).then(function(bundle) {
        return bundle.write({
            format: 'umd',
            dest: 'dist/js/myBundle.umd.js',
            moduleName: 'myBundle',
            globals: {
                'jQuery': 'jQuery'
            }
        });
    });
});
```

It works as expect 💯

**Comment by TrySound** [comment]
Yeah. skipped that one.

**Comment by eventualbuddha** [comment]
> This really sucks.Why we even need additional named export?

It's from before rollup supported the options hook. We should bump the major version and get rid of the other export.

**Comment by eventualbuddha** [comment]
Released in v2.0.0.
